### ISSUE TITLE
Cannot delete pod

### ISSUE THREAD

**Issue by Selenion** [kontena/lens#7835]
**Describe the bug**

When you trying to delete pod through Lens UI, nothing happens.
I tried different clusters with different kubeconfigs but the feature not work.
With same kubeconfigs deletion through console kubectl works fine.

**To Reproduce**
1. Go to Pods submenu in the Workloads
2. Click on pod
3. Click Recycle bin.
4. Press button "Remove"
5. Nothing happens

**Expected behavior**
Pod starting to terminate

**Environment (please complete the following information):**
- Lens Version:
Lens: 2023.5.310801-latest
Extension API: 6.5.2
Electron: 22.3.10
Chrome: 108.0.5359.215
Node: 16.17.1
- OS: [e.g. OSX]
macOS 13.4
- Installation method (e.g. snap or AppImage in Linux):
Download from https://k8slens.dev

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
No logs appear while I press the "Remove" button

**Comment by diegocejas** [comment]
I have the same exact issue as of today, tried by uninstalling/installing the app a couple of times to no avail.
Need a fix for this ASAP.

**Comment by panuhorsmalahti** [comment]
What is the Kubernetes version?

This might be due to the usage of the Eviction API for delete, but this is not supported in <1.22.

**Comment by Selenion** [comment]
> What is the Kubernetes version?
> 
> This might be due to the usage of the Eviction API for delete, but this is not supported in <1.22.

Kubelet version
v1.21.3-mirantis-1

**Comment by sachinshakya507** [comment]
Getting same issue :)
Lens-- lens/stable,now 2023.5.310801-latest amd64 [installed]

**Comment by Selenion** [comment]
I think it somehow related to https://github.com/lensapp/lens/issues/5602
It says that Lens use Eviction API only where it applicable, but seems not.

**Comment by nergalex** [comment]
Same issue.
K8S server: 1.23.17
Lens: 2023.5.310801-latest

**Comment by HunterSherms** [comment]
I am also suddenly having this issue, where I was not until recently.

**Comment by carn1x** [comment]
Same here, my cluster is k8s 1.21, using Lens-2023.5.310801-latest.x86_64.AppImage

**Comment by IAkumaI** [comment]
k8s 1.20.4

Same issue. Just nothing happens.

Jobs removing, for instance, works fine.

**Comment by SonicGD** [comment]
In our cluster users don't have access to Eviction API. So now we can't delete pods, even if it's already evicted. k8s throws error:

```
pods "pod-name" is forbidden: User "user-name" cannot create resource "pods/eviction" in API group "" in the namespace "namespace-name"
```
<img width="809" alt="image" src="https://github.com/lensapp/lens/assets/1222013/102edbd6-7af1-4263-b790-04a39c4a9c2a">

**Comment by nlypka** [comment]
Same issue, running EKS with 1.24 - Lens: 2023.5.310801-latest. Can confirm I am able to delete pods using kubectl and the same context+auth token.

**Comment by sachinshakya507** [comment]
> Getting same issue :) Lens-- lens/stable,now 2023.5.310801-latest amd64 [installed]

I have two config one with full access and other with rbac for same k8s-cluster.
I could easily remove my pods on both config before.
But somehow i face an issue from my rbac config only where i couldn't delete my pod anymore.
This was my rbac config before

apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  namespace: my_namespace
  name: my_name
rules:
- apiGroups: [""]
  resources: ["pods","pods/log",]
  verbs: ["get", "watch", "list","delete"]

What i added on my rbac so i could make my rbac config to work again 
- apiGroups: [""]
  resources: ["pods/eviction"]
  verbs: ["create", "delete"]

**Comment by jweak** [comment]
This will be fixed in a upcoming Lens Desktop release. Evict will have it's own menu item and Delete will be reverted to use `delete` API.

**Comment by MahnoorShafique** [comment]
Is the above issue resolved?
@jweak

**Comment by jweak** [comment]
> Is the above issue resolved?
> @jweak

It has been fixed but not released yet.

**Comment by MahnoorShafique** [comment]
@jweak When will this be released?

**Comment by M4RC0Sx** [comment]
> @jweak When will this be released?

Also waiting for it!

**Comment by nopjar** [comment]
> This will be fixed in a upcoming Lens Desktop release. Evict will have it's own menu item and Delete will be reverted to use `delete` API.

Seeing that such a breaking bug is fixed but not yet delivered for over three weeks is kinda disappointing.

Is there any ETA?

**Comment by mrnonz** [comment]
Should we release this patch first? I think this impact many users. Thanks

**Comment by MahnoorShafique** [comment]
@mrnonz yes please

**Comment by r97221004** [comment]
@mrnonz yes please!!

**Comment by chernykhSG** [comment]
@mrnonz yes please

**Comment by MBagrat** [comment]
@mrnonz yes please.

Our team (more than 10 people) is waiting for this issue to be fixed.

**Comment by IAkumaI** [comment]
@mrnonz yes, this is very important

**Comment by RightInTwo** [comment]
+1

**Comment by Selenion** [comment]
Looks like something not good is happening. There is no releases since 30 may.

**Comment by devosonder** [comment]
+1

**Comment by bognerf** [comment]
+1

**Comment by jim-docker** [comment]
The fix was deemed too large for a patch release so it it will come out in the next minor release, hopefully next week. Sorry for the inconvenience.

Workaround: Run `kubectl delete pod pod-name` in the Lens terminal. (Copy the pod-name from the the title of the pod's details page, paste into the lens terminal -- right-click if you've set that up in Preferences -> Terminal -> TERMINAL COPY & PASTE)   :(

**Comment by noskovao** [comment]
+1, please release ASAP, it is annoying to copy the pod name and go thru the kubectl :/

**Comment by MahnoorShafique** [comment]
Yes, copying every pod name and deleting it with the command is annoying.

**Comment by Selenion** [comment]
@jim-docker any updates please?

**Comment by qkflies** [comment]
New release out per their [forums](https://forums.k8slens.dev/t/lens-2023-8-030801-latest-release/709), but no new release in here on Github, no new commits and barely any activity for two months and counting. @jweak @jim-docker is OpenLens discontinued/abandoned?

**Comment by qixiaobo** [comment]
Same problem and several versions were OK before

**Comment by mrnonz** [comment]
> Same problem and several versions were OK before

Try to upgrade to version `2023.8.31525-latest`

**Comment by qixiaobo** [comment]
> > Same problem and several versions were OK before
> 
> Try to upgrade to version `2023.8.31525-latest`
@mrnonz 

Lens: 2023.5.310801-latest
Extension API: 6.5.2
Electron: 22.3.10
Chrome: 108.0.5359.215
Node: 16.17.1
© 2023 Mirantis, Inc.

And no update

You've got the latest version of Lens,
thanks for staying on the ball.

**Comment by jweak** [comment]
> > > Same problem and several versions were OK before
> > 
> > 
> > Try to upgrade to version `2023.8.31525-latest`
> > @mrnonz
> 
> Lens: 2023.5.310801-latest Extension API: 6.5.2 Electron: 22.3.10 Chrome: 108.0.5359.215 Node: 16.17.1 © 2023 Mirantis, Inc.
> 
> And no update
> 
> You've got the latest version of Lens, thanks for staying on the ball.

https://forums.k8slens.dev/t/lens-2023-8-31525-latest-release/709
> To ensure a smooth transition and minimize disruption, we have decided to implement this update in stages. We’ll start by rolling out the update to a small portion of our users, then incrementally increasing this number until all users have access to the update.

If you are not part of the group who are offered in-app update, you can download latest version directly from https://k8slens.dev

**Comment by qixiaobo** [comment]
> > > > Same problem and several versions were OK before
> > > 
> > > 
> > > Try to upgrade to version `2023.8.31525-latest`
> > > @mrnonz
> > 
> > 
> > Lens: 2023.5.310801-latest Extension API: 6.5.2 Electron: 22.3.10 Chrome: 108.0.5359.215 Node: 16.17.1 © 2023 Mirantis, Inc.
> > And no update
> > You've got the latest version of Lens, thanks for staying on the ball.
> 
> https://forums.k8slens.dev/t/lens-2023-8-31525-latest-release/709
> 
> > To ensure a smooth transition and minimize disruption, we have decided to implement this update in stages. We’ll start by rolling out the update to a small portion of our users, then incrementally increasing this number until all users have access to the update.
> 
> If you are not part of the group who are offered in-app update, you can download latest version directly from https://k8slens.dev

OK I will try
![image](https://github.com/lensapp/lens/assets/2881751/6fb911ce-370b-4d0b-9919-a547b12f5ba1)

**Comment by qixiaobo** [comment]
> > > > > Same problem and several versions were OK before
> > > > 
> > > > 
> > > > Try to upgrade to version `2023.8.31525-latest`
> > > > @mrnonz
> > > 
> > > 
> > > Lens: 2023.5.310801-latest Extension API: 6.5.2 Electron: 22.3.10 Chrome: 108.0.5359.215 Node: 16.17.1 © 2023 Mirantis, Inc.
> > > And no update
> > > You've got the latest version of Lens, thanks for staying on the ball.
> > 
> > 
> > https://forums.k8slens.dev/t/lens-2023-8-31525-latest-release/709
> > > To ensure a smooth transition and minimize disruption, we have decided to implement this update in stages. We’ll start by rolling out the update to a small portion of our users, then incrementally increasing this number until all users have access to the update.
> > 
> > 
> > If you are not part of the group who are offered in-app update, you can download latest version directly from https://k8slens.dev
> 
> OK I will try ![image](https://user-images.githubusercontent.com/2881751/260401248-6fb911ce-370b-4d0b-9919-a547b12f5ba1.png)

![image](https://github.com/lensapp/lens/assets/2881751/699e3100-d6b3-47ce-a987-5d38fd095f32)

This version works fine.
But why my old version not receive updates? Different release channel?

**Comment by panuhorsmalahti** [comment]
@qixiaobo we haven't rolled out the latest version to all users yet since we identified an issue with extensions. It's effectively random which users get the update at this point, but users can manually download the latest version from our website.

**Comment by qixiaobo** [comment]
> @qixiaobo we haven't rolled out the latest version to all users yet since we identified an issue with extensions. It's effectively random which users get the update at this point, but users can manually download the latest version from our website. 

Thanks for the reply. Maybe add some release channels to avoid this ?
